uber/models/__init__.py

```diff
@@ -1376,12 +1381,35 @@ def indie_games(self):
             return self.query(IndieGame).join(IndieStudio).options(
                 joinedload(IndieGame.studio), joinedload(IndieGame.reviews)).order_by(IndieStudio.name, IndieGame.title)
 
+        def create_or_find_mivs_judge_access_group(self):
+            """
+            Looks for an admin access group with write access to only mivs_judging,
+            and creates a new one if it can't find any.
+
+            Technically, we don't need this access group -- access to mivs_judging
+            is determined by whether the admin account is linked to an IndieJudge
+            object -- but we need to give MIVS judges some sort of access group.
```

Comment:
Interesting.  That seems inelegant but I can't think of a better solution.  Which is to say, the more general solutions are all a lot more complicated and involve a lot more code, so I that (surprisingly) this is probably the most elegant solution - good stuff.

I guess we can pay attention to how often we need this kind of ``get_or_create_xxx_access_group`` and if it ends up all over the place.

Reply:
Yea, luckily for us I think this is the only case where we make the code create an admin account. It really is a special case since MIVS Judges aren't _really_ admins, I don't think, but we make them admin accounts.